src/login/styles/_Nins.scss

```diff
@@ -82,14 +82,19 @@
   }
 }
 
+
 .proofing-btn-help {
   margin: 0 0.5rem 1rem 0.5rem;
   line-height: 1.2;
-  color: #7c7c7c;
+  color: $black;
   font-size: 0.875rem;
   width: 300px;
 }
 
+.proofing-btn-help.disabled {
+  color: #7c7c7c;
```

Comment:
are we using this colour a lot for disabled things? if so can it be turned into a variable?